firmware/thruna/DIM/test/test_avgPowerCalc.cpp

```diff
@@ -1,15 +1,7 @@
-#include <gtest/gtest.h>
-#include "Test_Utils.h"
+#include "test_dimBaseStateMachine.h"
 
-extern "C"
+class AveragePowerCalcTest : public DimBaseStateMachineTest
 {
-#include "app_avgPower.h"
-}
-
-class AveragePowerCalcTest : public testing::Test
-{
-  protected:
-    void SetUp() override { app_avgPower_init(); }
```

Comment:
previous AveragePwoerCalcTest only required the initialization of `app_avgPower_init()` rather than initializing the whole state machine, however to better fit the two tests relating to average power calculation together, needed the state machine anyways, and I think the slight performance overhead of initializing the state machine is worth having a cleaner file